src/game_renderer/GameRenderer.h

```diff
@@ -3,16 +3,39 @@
 #ifndef GAME_RENDERER_H_
 #define GAME_RENDERER_H_
 
+#include <cmath>
+#include <fstream>
+#include <glm/gtc/type_ptr.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+#include <glob.h>
+#include <iostream>
+#include <unordered_map>
 #include "game_state/GameState.h"
+#include "GLFW/glfw3.h"
+#include "GL/glew.h"
+#include "GLSL.h"
+#include "json.hpp"
+#include "MatrixStack.h"
+#include "Program.h"
+#include "Texture.h"
 
 class GameRenderer {
  public:
   GameRenderer();
   ~GameRenderer();
-
+  GLFWwindow* GetWindow();
+  std::shared_ptr<Program> ProgramFromJSON(std::string filepath);
+  std::vector<std::string> ListFiles(const std::string& pattern);
```

Comment:
I think at least ListFiles should be in its own class, maybe FileSystemUtils or something similar.